services-js/registry-certs/client/queries/graphql-types.ts

```diff
@@ -128,10 +128,20 @@ export interface SearchDeathCertificatesVariables {
 // GraphQL mutation operation: SubmitDeathCertificateOrder
 // ====================================================
 
-export interface SubmitDeathCertificateOrder_submitDeathCertificateOrder {
+export interface SubmitDeathCertificateOrder_submitDeathCertificateOrder_order {
```

Comment:
Sooo ugly! Thanks a lot, GraphQL. 😆

Reply:
Yah. This is why sometimes I just slip `type` aliases into `types.ts`.